app/build.gradle

```diff
@@ -122,6 +124,9 @@ dependencies {
     androidTestImplementation 'app.cash.turbine:turbine:0.7.0'
     androidTestImplementation "androidx.arch.core:core-testing:2.2.0"
     androidTestImplementation "androidx.work:work-testing:$work_version"
+    androidTestImplementation 'com.google.dagger:hilt-android-testing:2.44'
```

Comment:
What I like to do, is when I have libraries tied to the same version (ex: Hilt has like 5 libraries, all tied to 2.44), i like creating a variable for the version

So in the future, I can just change it in one place, and it gets changed everywhere